anitya/__init__.py

```diff
@@ -1,10 +1,19 @@
 # -*- coding: utf-8 -*-
 
+from __future__ import absolute_import
```

Comment:
It's not actually needed, I just haven't gone through to clean up things added by python-modernize that aren't strictly necessary - I'll add that to the TODO list.